src/block-components/block-div/firefox-has-polyfill.js

```diff
@@ -0,0 +1,33 @@
+
+export const firefoxHasPolyfill = ( clientId, attributes ) => {
+	if ( ! clientId ) {
+		return <></>
```

Comment:
It's better to return null here so that it doesn't have to go through `Fragment`

```suggestion
		return null
```